metadata/compute-results-2024-12-12T13-21-45-215Z.txt

```diff
@@ -0,0 +1 @@
+"Traceback (most recent call last):\r\n  File \"data/transformations/algorithm\", line 18, in <module>\r\n    \"termsAndConditions\": true\r\nNameError: name 'true' is not defined\r\n"
```

Comment:
in the ddo, termsAndConditions should be set to `True` if you choose py algo to run.

Reply:
I fixed this by setting the DDO to have `"termsAndConditions": "True"`